ui/js/controllers/logviewer.js

```diff
@@ -189,7 +189,8 @@ logViewerApp.controller('LogviewerCtrl', [
                     for (let i = 0; i < $scope.steps.length; i++) {
                         let step = $scope.steps[i];
 
```

Comment:
When I put a console.log() statement here to see if I could use the job result status, this whole logviewerInit() function appears to get called twice. Once where $scope.result is undefined, and another time where it is defined. Hopefully this all is working correctly.